github-pull-requests.go

```diff
@@ -386,34 +79,27 @@ func populatePRs(url string) {
 	htmlData, err := ioutil.ReadAll(resp.Body)
 	exitOnErr(err)
 
+	const layout = "Jan 2, 2006 at 3:04pm (PST)"
```

Comment:
Do you want this to be a custom layout?  doesn't http.TimeFormat or time.RFC1123 serve the same purpose?